test_data/stdlib/2and3/collections_test.py

```diff
@@ -0,0 +1,15 @@
+from collections import namedtuple
+
+
+def test_namedtuple():
+    Point = namedtuple('Point', 'x y')
+    p = Point(1, 2)
+
+    assert p == Point(1, 2)
```

Comment:
Note that mypy currently can't check that the return type is correct here. For example, if we'd declare `def f() -> int: ...` and had a test like `assert f() == 'x'` mypy wouldn't flag it as an error (see https://github.com/python/typing/issues/378 and https://github.com/python/mypy/issues/1271). Would pytype flag this as an error? 

Even if we can't check these yet, the assertions are useful for code reviewers, as it makes it easier to manually check that a return type annotation looks reasonable.